### PR TITLE
Use mania-specific judgement sprites for mania

### LINKED ISSUE #7324: Mania should use mania-specific judgement sprites

### PR DESCRIPTION
**Pull request by mcendu**
Fixes #7324

Prereqs:
- [x] ppy/osu-resources#83

### CODE REVIEW

**Review by mcendu — COMMENTED** [review]
Legacy-like animation still needs to be implemented.

File: osu.Game.Rulesets.Mania/Skinning/ManiaLegacySkinTransformer.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+                        case HitResult.Good:
+                            return this.GetAnimation("mania-hit200", true, true);
+
+                        case HitResult.Great:
+                            return this.GetAnimation("mania-hit300", true, true);
+
+                        case HitResult.Perfect:
+                            return this.GetAnimation("mania-hit300g", true, true);
+                    }
```
Comment (bdach) [inline osu.Game.Rulesets.Mania/Skinning/ManiaLegacySkinTransformer.cs]: Nitpicky, but maybe have this as a separate method?

**Review by bdach — COMMENTED** [review]
Source itself looks reasonable to me.

**Comment by peppy** [comment]
I think this is fine as a first step. Animation speed isn't correct and animation doesn't match 1:1, but this can be done as a follow-up.

**Review by peppy — DISMISSED** [review]

**Review by peppy — APPROVED** [review]

**Comment by peppy** [comment]
I also think the size needs adjustment. Will need an accuracy pass, comparing playfield with stable (via screenshots or video).

**Pull request merged**